### ISSUE TITLE
ng-jq on <html> gives error

### ISSUE THREAD

**Issue by alexlamsl** [angular/angular.js#11016]
So I follow the example in the documentation to try `ng-jq` out, and as far as I can tell, `getNgAttribute()` is called within `jq()`, which in turn uses `jqLite`, which hasn't been defined yet.

**Comment by Narretz** [comment]
Hi, what kind of error is thrown?

**Comment by alexlamsl** [comment]
Tested on `angular.js 1.4.0-beta.4` and IE11, using example HTML:
```
<html ng-app="app" ng-jq>
...
</html>
```
gives the following within `Developer Tools`:
```
'jqLite' is undefined
```
showing the reference inside `function getNgAttribute()`

**Comment by Narretz** [comment]
Yep, that's broken: http://plnkr.co/edit/dRlqiPdHKU6xchecBVnE?p=preview

@mboudreau @lgalfaso could you look into this?

**Comment by mboudreau** [comment]
@alexlamsl @Narretz Yeah, seems broken.  The getNgAttribute was added later as per comments on the pull request.  What's weird is that this isn't caught in the unit tests.  I would of imagined they would of failed entirely in the unit tests, but I guess by that point, Angular is already loaded and jqLite already initialized.

Let me fix this and put in a new pull request ASAP.

**Comment by mboudreau** [comment]
@alexlamsl @Narretz It's been taken cared of, shouldn't cause any issues, but definitely needs to add better tests around it.  Just waiting for the pull request to be merged in and hopefully go out in the next beta revision.

Thanks for letting me know on the issue.

**Comment by petebacondarwin** [comment]
This fix was landed via #11044 - see http://plnkr.co/edit/Xa7P4JI0EAJtnVwDS5mX?p=preview

**Comment by mboudreau** [comment]
@petebacondarwin You da man.
